views/bonfire/show.jade

```diff
@@ -143,5 +149,9 @@ block content
                     $('.btn-twitter').attr('href', url);
                 }
         );
+        var MDNlinks = !{JSON.stringify(MDNlinks)};
```

Comment:
@BerkeleyTrue I get what `JSON.stringify(MDNlinks)` does, but thanks for clarifying that the `!{...}` syntax is from Jade.  I'll look into that.